src/Service/NotificationReceiverService.php

```diff
@@ -273,6 +273,7 @@ protected function processNotificationItem($notificationItem)
                 // duplicated so do nothing but return accepted to Adyen
                 $this->logger->info('Notification is a TEST notification from ' .
```

Comment:
Is this message correct? Is it duplicated or test notification?

Reply:
It is duplicated, will change the message